frontend/packages/kubevirt-plugin/integration-tests/tests/vm.resources.scenario.ts

```diff
@@ -178,9 +166,9 @@ describe('Test network type presets and options', () => {
     expect((await getSelectOptions(nicType)).sort()).toEqual(nonPodNetworkBindingMethods);
   });
 
-  it('ID(CNV-4038) Test NIC default type in example VM', async () => {
+  xit('BZ(1828739) ID(CNV-4038) Test NIC default type in example VM', async () => {
```

Comment:
Do we not want this test to run?

Reply:
I'm skipping the test intentionally due to the opened bug